water/include/water/Dialect/Wave/Transforms/LoweringPatterns.h

```diff
@@ -68,6 +68,10 @@ void populateWaveReadWriteLoweringPatterns(WaveTypeConverter &typeConverter,
 void populateWaveUnaryFPOpLoweringPatterns(WaveTypeConverter &typeConverter,
                                            mlir::RewritePatternSet &patterns);
 
+// Adds pattern that lowers 'wave.extract_slice' ops to upstream MLIR ops.
+void populateWaveExtractSliceLoweringPatterns(
+    WaveTypeConverter &typeConverter, mlir::RewritePatternSet &patterns);
```

Comment:
Can we please stop adding a function per operation? Just put it in misc lowering above...